src/babel-plugin-bare-specifiers.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * @license
+ * Copyright (c) 2018 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {dirname, relative} from 'path';
+import resolve = require('resolve');
+import {NodePath} from 'babel-traverse';
+import {ImportDeclaration, ExportNamedDeclaration, ExportAllDeclaration} from 'babel-types';
+
+const exportExtensions = require('babel-plugin-syntax-export-extensions');
+
+const isPathSpecifier = (s: string) => /^\.{0,2}\/.*/.test(s);
```

Comment:
the trailing `.*` could be removed, it's a noop

Reply:
done